src/commands/kol/player.ts

```diff
@@ -0,0 +1,84 @@
+import {
+    ChatInputCommandInteraction,
+    SlashCommandBuilder,
+    bold, 
+    hyperlink
+  } from "discord.js";
+  
+import { kolClient } from "../../clients/kol";
+import { pluralize, toKoldbLink, toSnapshotLink } from "../../utils";
+
+
+export const data = new SlashCommandBuilder()
+  .setName("player")
+  .setDescription("Look up information on a given player.")
+  .addStringOption((o) =>
+    o
+      .setName("player")
+      .setDescription("The KOL username of the player you're looking up.")
+      .setRequired(true)
+    );
+
+export async function execute(interaction: ChatInputCommandInteraction) {
+    const member = interaction.member;
+  
+    if (!member) {
+      interaction.reply({
+        content: "You have to perform this action from within a Guild.",
+        ephemeral: true,
+      });
+      return;
+    }
```

Comment:
This feels like a recurring bit of logic we might want to slice out into its own function

Reply:
maybe; in this case you could argue we don't really NEED this to be something cast from ASS vs DMs, but i just feel like it's a neat thing in a forum context and kind of weird if we have people just constantly DMing oaf for weird people links. i think it's currently in 3-4 places so i'm not sure it quite hits the "this should be a util function" yet but it's close for sure.

Reply:
As long as that util function is also a typeguard on member I agree

Reply:
Hang on why are you limiting this to being in a guild?? Just remove the whole chunk